paddle/fluid/framework/ir/mkldnn/cpu_quantize_pass.cc

```diff
@@ -676,6 +677,79 @@ void CPUQuantizePass::QuantizeMatmul(Graph* graph) const {
   PrettyLogDetail("---    quantized %d matmul ops", quantize_matmul_count);
 }
 
+void CPUQuantizePass::QuantizeElementwiseAdd(Graph* graph) const {
+  GraphPatternDetector gpd;
+  auto pattern = gpd.mutable_pattern();
+  patterns::ElementwiseAdd elementwise_add_pattern{pattern, name_scope_};
+
+  elementwise_add_pattern(
+      pattern->NewNode(elementwise_add_pattern.elementwise_add_x_repr()),
+      pattern->NewNode(elementwise_add_pattern.elementwise_add_y_repr()));
+
+  int quantize_elementwise_add_count = 0;
+  auto handler = [&](const GraphPatternDetector::subgraph_t& subgraph,
+                     Graph* g) {
+    VLOG(4) << "Quantize elementwise_add op";
+    GET_IR_NODE_FROM_SUBGRAPH(elementwise_add_op, elementwise_add_op,
+                              elementwise_add_pattern);
+    auto* elementwise_add_op_desc = elementwise_add_op->Op();
+
+    // skip if should not be quantized
+    if (!elementwise_add_op_desc->GetAttrIfExists<bool>("use_quantizer")) {
+      return;
```

Comment:
Please add a VLOG like info why it was skipped

Reply:
I've seen other quantization passes in this file and all of them don't have a VLOG. I think we should either put VLOGs in all of them or not. Should I add it to all passes in cpu_quantize_pass @arlesniak ? @wojtuss what do you think?

Reply:
I referred to line 692 with VLOG saying "Quantize ..." suggesting the quantization will be applied. But here (line 699) we skip the next steps silently by return. I could suggest that moving the VLOG message from 692 to 701 would be sufficient to give clear message to the user.

Reply:
or adding LogCannotQuantizeOp with explanation. What do you think ?

Reply:
As far as I know, `use_quantizer` is is in `cpu_quantize_placement_pass` to all operators that have that attribute (in other words are quantizable), unless they are explicitly excluded/not-included from command-line. I think that this is sufficiently self-explanatory that we do not quantize such operators so personally I think we can skip it because there is not much added value.

Reply:
I agree that adding the VLOG could be done here in a general manner, i.e. in other places as well.
To me the "Quantize ..." log says only that quantization happens, but at the end of the pass we report how many operators were actually quantized.